core/domain/rule_domain_test.py

```diff
@@ -99,10 +97,48 @@ def test_that_all_rules_have_object_editor_templates(self):
                         param_obj_type.has_editor_js_template(),
                         msg='(%s)' % clazz.description)
                     at_least_one_rule_found = True
-                clses.append(clazz)
 
         self.assertTrue(at_least_one_rule_found)
 
+    def test_that_all_rule_input_fields_have_default_values(self):
+        rule_dir = os.path.join(os.getcwd(), feconf.RULES_DIR)
+
+        at_least_one_rule_found = False
```

Comment:
what about instead of at_least_one_rule_found -> number_of_rules, assertEquals(16, number_of_rules) ?